native/copyNativeLibs.ts

```diff
@@ -0,0 +1,112 @@
+// Heavily inspired by napi-rs's "napi" script, (https://github.com/napi-rs/napi-rs/blob/master/scripts/napi.js) 
```

Comment:
This entire goddamn file is a glorified copy/rename

Reply:
Okay, it's a little better now. I tore out the stuff we don't use, and its defaults are ones we want.